### PR TITLE
gptp: fixing/adding logging

### PR DESCRIPTION
**Pull request by PawelModrzejewski**
Deatils:
    - changed fprintf to proper error log message;
    - error log messege added to avoid silent passing
      Sync/FollowUp processing;

### CODE REVIEW

**Comment by andrew-elder** [comment]
Looks good. Thanks. Merging....

File: daemons/gptp/common/ptp_message.cpp [hunk 1]
```diff
@@ -1037,6 +1037,7 @@ void PTPMessageFollowUp::processMessage
 	port->incCounter_ieee8021AsPortStatRxFollowUpCount();
 
 	if (!port->getLinkDelay(&delay))
+		GPTP_LOG_ERROR("Received Follow up but there is no valid link delay");
```
[the remark this answers is not in the document]
Comment (andrew-elder) [reply daemons/gptp/common/ptp_message.cpp]: Good catch Chris. Who wants to fix it?

[the remark this answers is not in the document]
Comment (PawelModrzejewski) [reply daemons/gptp/common/ptp_message.cpp]: Monday ...

[the remark this answers is not in the document]
Comment (andrew-elder) [reply daemons/gptp/common/ptp_message.cpp]: I created a new pull request with a patch.

**Pull request merged**